lang/en.js

```diff
@@ -7,6 +7,7 @@ export default {
 	"components.calendar.selected": "Selected.",
 	"components.calendar.show": "Show {month}",
 	"components.dialog.close": "Close this dialog",
+	"components.filter.filters": "Filters",
```

Comment:
The more complicated langterm discussed will be coming once I add the ability to count and show num selected (future story)